upup/pkg/fi/cloudup/vsphere/vsphere_cloud.go

```diff
@@ -36,6 +40,18 @@ func (c *VSphereCloud) ProviderID() fi.CloudProviderID {
 	return fi.CloudProviderVSphere
 }
 
+func NewVSphereCloud(spec *kops.ClusterSpec) (*VSphereCloud, error) {
+	server := spec.VSphereServer
+	datacenter := spec.VSphereDatacenter
+	cluster := spec.VSphereResourcePool
+	username := os.Getenv("VSPHERE_USERNAME")
+	password := os.Getenv("VSPHERE_PASSWORD")
+
```

Comment:
Not a problem for now, since we don't really create a client yet. But if the flags are not set, there will be an error jumping out earlier. Now it could be just some dummy values.